hilla-shaded-deps/pom.xml

```diff
@@ -50,6 +50,7 @@
                                         <include>org/springframework/core/io/InputStreamSource.class</include>
                                         <include>org/springframework/core/io/Resource.class</include>
                                         <include>org/springframework/core/io/ResourceLoader.class</include>
+                                        <include>org/springframework/lang/NonNullApi.class</include>
```

Comment:
Now shaded?

Reply:
Yes. The replacement works during quarkus build, but it doesn't for the Hilla production build.
So we need the original spring class

Reply:
Can this be fixed? Is the own NonNull annotation still valid?

Reply:
Unfortunately, the extension can't do anything for the Hilla production build.
One should configure the maven plugin manually to configure the nonnull Hilla plugin.